Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -13,6 +13,23 @@
 // This file provides compatibility aliases to keep dependents of SwiftSyntax building.
 // All users of the declarations in this file should transition away from them ASAP.
 
+public extension AccessorEffectSpecifiersSyntax {
+  @available(*, deprecated, message: "use throwsClause instead of throwsSpecifier")
+  init(
+    leadingTrivia: Trivia? = nil,
+    asyncSpecifier: TokenSyntax? = nil,
+    throwsSpecifier: TokenSyntax? = nil,
+    trailingTrivia: Trivia? = nil
+  ) {
```

Comment:
These convenience initializers need to also take unexpected nodes. Their API signature should match the initializers that we previously had.